app/src/main/java/org/simple/clinic/main/TheActivityComponent.kt

```diff
@@ -132,6 +134,7 @@ interface TheActivityComponent :
   fun inject(target: EditPatientScreen)
   fun inject(target: SettingsScreen)
   fun inject(target: ChangeLanguageScreen)
+  fun inject(target: DeletePatientScreenInjector)
```

Comment:
Why is this needed? There's no point in injecting an interface, right?

Reply:
Oh wow, I have no idea why I added that 😅 . It's a mistake, removing it.